shell/platform/embedder/embedder.cc

```diff
@@ -92,7 +108,17 @@ static bool IsOpenGLRendererConfigValid(const FlutterRendererConfig* config) {
 
   return true;
 }
+#endif  // SHELL_ENABLE_GL
+
+#if SHELL_ENABLE_VULKAN
```

Comment:
When using Vulkan, we don't really need to link to libVulkan.so since proc resolution is performed at runtime. I say it makes sense to make all embedder dylibs Vulkan aware. To that end, we'd need to offload Vulkan proc resolution to the embedder.